girder_worker/core/tests/CMakeLists.txt

```diff
@@ -10,23 +10,23 @@ if(PYTHON_COVERAGE)
   add_test(
     NAME py_coverage_combine
     WORKING_DIRECTORY "${PROJECT_SOURCE_DIR}"
-    COMMAND "${PYTHON_COVERAGE_EXECUTABLE}" combine
+    COMMAND "${PYTHON_COVERAGE_EXECUTABLE}" combine "--rcfile=${py_coverage_rc}"
   )
   add_test(
     NAME py_coverage
     WORKING_DIRECTORY "${PROJECT_SOURCE_DIR}"
-    COMMAND "${PYTHON_COVERAGE_EXECUTABLE}" report --fail-under=${COVERAGE_MINIMUM_PASS}
+    COMMAND "${PYTHON_COVERAGE_EXECUTABLE}" report "--rcfile=${py_coverage_rc}" --fail-under=${COVERAGE_MINIMUM_PASS}
```

Comment:
fail-under could be moved,  but the rcfile now has to point to the ```{build}/girder_worker/core/tests/girder_worker.coveragerc``` file otherwise it tries to report from the 'default' data file (.coverage)  which no longer exists.